internal/mcp_server.go

```diff
@@ -158,17 +168,17 @@ func (s mcpServer) RegisterToolsAndResources() error {
 			mcp.Required(),
 		),
 	)
-	s.server.AddTool(t, s.ReplayTool)
+	s.server.AddTool(t, mcp.NewTypedToolHandler(s.ReplayTool))
 
 	return nil
 }
 
-func (s mcpServer) Start() error {
-	slog.Info("Starting Nobl9 MCP server", "version", "0.1.0")
+func (s *mcpServer) Start() error {
```

Comment:
we have a mixed pointer and value receivers. I think we can just stick to value receivers.